gtwrap/interface_parser/classes.py

```diff
@@ -225,21 +225,6 @@ def __repr__(self) -> str:
         )
 
 
-def collect_namespaces(obj):
-    """
-    Get the chain of namespaces from the lowest to highest for the given object.
-
-    Args:
-        obj: Object of type Namespace, Class or InstantiatedClass.
-    """
-    namespaces = []
-    ancestor = obj.parent
-    while ancestor and ancestor.name:
-        namespaces = [ancestor.name] + namespaces
-        ancestor = ancestor.parent
-    return [''] + namespaces
-
-
 class Class:
```

Comment:
You're not allowing enums in classes?

Reply:
Not currently. We can add an issue for this.